### PR TITLE
Add 'go to core/install.php' to README.txt and add developer friendly settings to the defaullt. local.settings.php

### PR DESCRIPTION
**Pull request by nuez**
I've added some developer friendly options to default.settings.local.php (from example.settings.local.php), assuming that local is used for development. Also completed the README.md to point out that users have to go to core/install.php (index.php throws an error when using a settings.php with an empty DB, see https://www.drupal.org/node/728702

### CODE REVIEW

File: web/sites/default/default.settings.local.php [hunk 1]
```diff
@@ -22,3 +22,26 @@
 
 // If you need to test the newsletter widget, set your Mailchimp API key below.
 $config['mailchimp.settings']['api_key'] = 'foo';
+
+/**
+ * Set development friendly settings and configurations.
+ *
+ * @see sites/example.settings.local.php
+ */
+assert_options(ASSERT_ACTIVE, TRUE);
```
Comment (juampynr) [inline web/sites/default/default.settings.local.php]: What's this for?

File: web/sites/default/default.settings.local.php [hunk 2]
```diff
@@ -22,3 +22,26 @@
 
 // If you need to test the newsletter widget, set your Mailchimp API key below.
 $config['mailchimp.settings']['api_key'] = 'foo';
+
+/**
+ * Set development friendly settings and configurations.
+ *
+ * @see sites/example.settings.local.php
```
Comment (juampynr) [inline web/sites/default/default.settings.local.php]: This feels like the file header, which we already have, don't you think?

File: web/sites/default/default.settings.local.php [hunk 3]
```diff
@@ -22,3 +22,26 @@
...
+$config['system.performance']['css']['preprocess'] = FALSE;
+$config['system.performance']['js']['preprocess'] = FALSE;
+
+// Disable render cache. (Do not change until after the site is installed.)
+# $settings['cache']['bins']['render'] = 'cache.backend.null';
+
+// Disable Dynamic Page cache.
+# $settings['cache']['bins']['dynamic_page_cache'] = 'cache.backend.null';
+
+$settings['extension_discovery_scan_tests'] = TRUE;
```
Comment (juampynr) [inline web/sites/default/default.settings.local.php]: Can you describe what each of these settings do?

**Comment by nuez** [comment]
All the explanations you can find in sites/example.settings.local.php, it's litteraly a copy past of that but without the comments. I don't understand everything in detail but quickly:

1. assert_options : A verbose output of Runtime Assertions used in Drupal
2. development.services.yml: Creates a 'Null' service for backend Cache.
3. error_level: verbose error levels.
4. Preprocess CSS/JS: Disable aggregation (easier debugging CSS / JS)
5. Render Cache - Set Backend Cache to NULL (created in 2.)
6. Dynamic Page Cache  - Set backend cache to NULL (created in 2)
7. rebuild_access: When accessing rebuild.php, cached storage will be rebuild.
8.skip_permissions_hardening. stop checking file permissions.

Basically these settings make debugging easier, as it deactivates a bunch of caching layers, and makes error reporting verbose.
